docs/src/main/sphinx/language/types.rst

```diff
@@ -262,11 +262,26 @@ The following examples illustrate the behavior::
 Instant in time that includes the date and time of day with ``P`` digits of
 precision for the fraction of seconds and with a time zone. Values of this
 type are rendered using the time zone from the value.
-Time zones are expressed as the numeric UTC offset value::
+Time zones can be expressed in the following ways:
+
+* ``UTC``, with ``GMT``, ``Z``, or ``UT`` usable as aliases for UTC.
+* ``+hh:mm`` or ``-hh:mm`` with ``hh:mm``, with or without ``UTC`` or an alias,
+  as an hour and minute offset from UTC.
+* An `IANA time zone name <https://www.iana.org/time-zones>`_.
+
+The following examples demonstrate these syntax options::
+
+    SELECT TIMESTAMP '2001-08-22 03:04:05.321 UTC';
+    -- 2001-08-22 03:04:05.321 UTC
+
+    SELECT TIMESTAMP '2001-08-22 03:04:05.321 -08:30';
+    -- 2001-08-22 03:04:05.321 -08:30
 
-    TIMESTAMP '2001-08-22 03:04:05.321 -08:00';
-    -- 2001-08-22 03:04:05.321-08:00
+    SELECT TIMESTAMP '2001-08-22 03:04:05.321 UTC -08:30';
+    -- 2001-08-22 03:04:05.321 UTC -08:30
```

Comment:
What does this query print when you run it in your Trino?

Reply:
Thanks for pointing this out. The space after `UTC` is errant so it should just be `UTC-8:30` in the SELECT statement.

The output is `2001-08-22 03:04:05.321 -08:00` regardless of the UTC alias, Fixing.